internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/job.go

```diff
@@ -10,13 +10,17 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/ctxattr"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/plugin"
+	keboolasink "github.com/keboola/keboola-as-code/internal/pkg/service/stream/sink/type/tablesink/keboola"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/model"
 )
 
-func (b *Bridge) createJob(ctx context.Context, token string, file plugin.File, storageJob *keboola.StorageJob) error {
+func (b *Bridge) SinkToken(ctx context.Context, sinkKey key.SinkKey) (keboolasink.Token, error) {
```

Comment:
Mhmm... I see why you didn't like this. I don't like it either.

What do you think about adding a `Cleanup` method to the bridge instead? I'm suggesting moving this entire block + cleanupJob into the bridge and simply calling bridge cleanup from `metacleanup.go`.

https://github.com/keboola/keboola-as-code/blob/a376096489a627100480d43bb6bdc64711af8fbb/internal/pkg/service/stream/storage/metacleanup/metacleanup.go#L174-L192

Reply:
Possible second step is to use the plugin system for cleanup and simply call `n.plugins.Cleanup(ctx)` from `metacleanup.go`.